spec/support/shared/shopify_request.rb

```diff
@@ -14,18 +14,17 @@
     ShopifyAPI::Base.user = nil
   end
 
-  def mock_request(endpoint, extension)
+  def mock_request(file, endpoint, extension)
```

Comment:
maybe we check if it's an array or not? What's transactions in the case of above?